ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -336,8 +350,15 @@ private void updateSubviewClipStatus(View subview) {
 
     // do fast check whether intersect state changed
     sHelperRect.set(subview.getLeft(), subview.getTop(), subview.getRight(), subview.getBottom());
-    boolean intersects = mClippingRect
-        .intersects(sHelperRect.left, sHelperRect.top, sHelperRect.right, sHelperRect.bottom);
+    Matrix matrix = subview.getMatrix();
+    if (!matrix.isIdentity()) {
```

Comment:
I was thinking of something like
``` java
if (view instanceof ReactClippingViewGroup) {
  ((ReactClippingViewGroup) view).updateClippingRect();
}
```
in https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java#L53